src/test/java/org/owasp/esapi/configuration/EsapiPropertyManagerTest.java

```diff
@@ -1,17 +1,22 @@
 package org.owasp.esapi.configuration;
 
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertNotSame;
+import static junit.framework.Assert.fail;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.owasp.esapi.ESAPI;
 import org.owasp.esapi.configuration.consts.EsapiConfiguration;
 import org.owasp.esapi.errors.ConfigurationException;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.FileNotFoundException;
-
-import static junit.framework.Assert.*;
-
 public class EsapiPropertyManagerTest {
```

Comment:
This test was modifying the system properties, and I was having downstream tests fail because the value of 'noSuchFile' (line 26) was failing to resolve.

This block of code ensures that when the test completes it restores the original property values to prevent corruption in downstream tests.